capstone/src/main/webapp/script.js

```diff
@@ -316,20 +306,28 @@ async function averagePrecinctSentiment(policePrecinct) {
     }
   }
   const sentimentAverage = sentimentCount/list.length;
-  return getSentimentColor(sentimentAverage);
+  return getSentimentColor(sentimentAverage, policePrecinct);
 }
 
-function getSentimentColor(averageFeelings) {
+function getSentimentColor(averageFeelings, policePrecinct) {
+  let precinctColor = '#228B22';
   if (Math.round(averageFeelings) == 5) {
-    return '#165B33';
+    precinctColor = '#165B33';
   } else if (Math.round(averageFeelings) == 4) {
-    return '#146B3A';
+    precinctColor = '#146B3A';
   } else if (Math.round(averageFeelings) == 3) {
-    return '#F8B229';
+    precinctColor = '#F8B229';
   } else if (Math.round(averageFeelings) == 2) {
-    return '#EA4630';
+    precinctColor = '#EA4630';
   } else {
-    return '#BB2528';
+    precinctColor = '#BB2528';
+  }
+  const precinctLayer = mapAndSelection.map;
+
+  if (policePrecinct!='Richmond'&& policePrecinct!='Southern') {
```

Comment:
The spacing here:

`if (policePrecinct != 'Richmond' && policePrecinct != 'Southern') {`

https://google.github.io/styleguide/jsguide.html#formatting-horizontal-whitespace

Also, do you mean `policePrecinct != 'Richmond' || policePrecinct != 'Southern'`? The current statement will always be true.

Reply:
Yes, thank you.

Reply:
Wait, I think that I need it to be && because if one is false I need it to return false

Reply:
If it were && the conditional would never evaluate to true since the String policePrecinct cannot hold two values at the same time